variables.tf

```diff
@@ -79,3 +79,62 @@ variable "server_name" {
   description = "The name of the server"
   default     = "default"
 }
+
+variable "cluster_ingress_hostname" {
```

Comment:
Can we remove this variable? Are you using the ingress hostname for something?

Reply:
implemented the change